js/screens/CommunityQuestion/CommunityQuestionContainer.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import CommunityQuestion from "./CommunityQuestion";
 
-export default class CommunityQuestionContainer extends Component {
+class CommunityQuestionContainer extends Component {
   constructor(props) {
     super(props);
   }
```

Comment:
remove the constructor if you don't need it.

Reply:
it has a proptype of navigation, I added it and kept the constructor and props